libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1447,6 +1468,51 @@ string YulUtilFunctions::readFromStorageDynamic(Type const& _type, bool _splitFu
 		.render();
 	});
 }
+string YulUtilFunctions::readFromStorageReferenceType(Type const& _type)
+{
+	solAssert(_type.category() == Type::Category::Struct, "");
```

Comment:
```suggestion
	solUnimplementedAssert(_type.category() == Type::Category::Struct, "");
```
This will handle arrays as well sooner or later, won't it?